Changes in module 'nf-core/custom/dumpsoftwareversions'
--- modules/nf-core/custom/dumpsoftwareversions/main.nf
+++ modules/nf-core/custom/dumpsoftwareversions/main.nf
@@ -1,11 +1,8 @@
 process CUSTOM_DUMPSOFTWAREVERSIONS {
     label 'process_single'

-    // Requires `pyyaml` which does not have a dedicated container but is in the MultiQC container
-    conda "bioconda::multiqc=1.14"
-    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/multiqc:1.14--pyhdfd78af_0' :
-        'biocontainers/multiqc:1.14--pyhdfd78af_0' }"
+    // Requires `pyyaml` which does not have a dedicated container but is in the crispr_pooled_screen container
+    label 'image_crispr_pooled_screen'

     input:
     path versions

************************************************************
